accesstoken2/src/models/request_payload.ts

```diff
@@ -0,0 +1,9 @@
+export class RequestPayload {
```

Comment:
The Java in you is here 👀 

This should likely be in the `types` directory and shouldn't be a class. It should be a Type or Interface instead. I think it should be a type.

```
export type RequestPayload {
  grantType: string;
  code: string:
....
}

Reply:
I wanted to return a new `RequestPayload` object

```
 return new RequestPayload(grant_type, code, redirectUri, client_assertion_type, client_assertion);
```

So I can keep the `URLSearchParams` out of the handler and work with it like so
```
requestPayload.code
requestPayload.clientAssertion
```

Reply:
You can do that by doing the following with a type - 

```
const myReturnedObject: RequestPayload = {
    grantType: grant_type,
    code,
    redirectUri,
    clientAssertionType: client_assertion_type,
    clientAssertion: client_assertion
    
};
return myReturnedObject
```